services/mail.js

```diff
@@ -1,4 +1,5 @@
 const mailClient = require('../mail/sendgrid.js')
+const newEmail = require('../mail/emailTemplate.js')
```

Comment:
[nit] it's not clear that `newEmail` is a class with 2 functions. I think `emailTemplate.newUser` makes slightly more sense than `newEmail.newUser`. Even better would be to rename the functions to `buildNewUserEmail` or something, and you can just import the function directly `const { buildNewUserEmail } = require('..')`